### PR TITLE
Fix order line total price

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it fixes the order line total price.

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-398 User should see the correct total price for cart lines](https://app.clickup.com/t/2549495/SALEOR-398)

**Review by dominik-zeglen — APPROVED** [review]

File: src/userAccount/views/OrderDetails/Page.tsx [hunk 1]
```diff
@@ -26,12 +26,12 @@ const extractOrderLines = (lines: OrderDetail_lines[]): ILine[] => {
         ...line.unitPrice,
         currency: line.unitPrice.currency,
         gross: {
-          amount: line.quantity * line.unitPrice.gross.amount,
           ...line.unitPrice.gross,
+          amount: line.quantity * line.unitPrice.gross.amount,
```
Comment (krzysztofwolski) [inline src/userAccount/views/OrderDetails/Page.tsx:30]: It should be taken from API

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: src/userAccount/views/OrderDetails/Page.tsx (hunk 1 above)
Comment (orzechdev) [reply src/userAccount/views/OrderDetails/Page.tsx]: Oh, I didn't know it is available in API. Fixed - update in https://github.com/mirumee/saleor-sdk/pull/32

Comment (orzechdev) [reply src/userAccount/views/OrderDetails/Page.tsx]: I know that it needs sdk dependency update, but sdk with update is not yet released

**Review by AlicjaSzu — APPROVED** [review]

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**